doc/myopic.rst

```diff
@@ -6,13 +6,13 @@ Myopic transition path
 
```

Comment:
Maybe change the section title to "Myopic foresight scenarios" to be consistent with "Overnight (greenfield) scenarios" and "Perfect foresight scenarios"